ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/BasicAction.java

```diff
@@ -3732,6 +3732,8 @@ else if (record.value() instanceof ExceptionDeferrer)
 
     //    private Mutex _lock = new Mutex(); // TODO
     private List<Throwable> deferredThrowables = new ArrayList<>();
+
+    protected boolean dynamic1PC = TxControl.dynamic1PC;
```

Comment:
The thing is, we need to make sure subordinates don't employ this facility - I guess we could introduce a boolean subordinate flag that is also checked?